src/components/MoneyRequestHeader.js

```diff
@@ -169,7 +170,7 @@ const MoneyRequestHeader = (props) => {
                     <SettlementButton
                         currency={props.report.currency}
                         policyID={props.report.policyID}
-                        shouldShowPaypal={false}
+                        shouldShowPaypal={shouldShowPaypal}
```

Comment:
this was most likely an oversight, this is same button just on mobile or web so not sure why the logic for showing paypal should be different in each